ptsprojects/zephyr/gatt_wid.py

```diff
@@ -140,26 +149,7 @@ def hdl_wid_17(desc):
         logging.error("%s parsing error", hdl_wid_17.__name__)
         return False
 
-    iut_services = []
-
-    # Get all primary services
-    attrs = btp.gatts_get_attrs(type_uuid='2800')
-    for attr in attrs:
-        handle, perm, type_uuid = attr
-        (_, uuid_len, uuid) = btp.gatts_get_attr_val(handle)
-        uuid = btp.btp2uuid(uuid_len, uuid)
-        iut_services.append(uuid)
-
-    # Verification
-    for service in pts_services:
-        if service in iut_services:
-            iut_services.remove(service)
-            logging.debug("Service %s found", service)
-            continue
-        else:
-            logging.error("Service %s not found", service)
-            return False
-    return True
+    return btp.verify_desc_values(pts_services)
```

Comment:
I guess it could break the verification for server side test cases that use this WID.
GATT/GATT/SR/GPA/BV-01-C,
GATT/GATT/SR/GAD/BV-01-C

[1] I suggest you to add

def hdl_wid_17(desc):
    return btp.verify_description(desc)

to gattc_wid.py